res/TensorFlowTests/NET_YUV_TO_RGB/circle_1.0_rel_requirement.rule

```diff
@@ -0,0 +1,7 @@
+# tf2circle-dredd-pbtxt-test for YUV to RGB conversion
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1  # 1 means verification is OK
+
+# RULE    "ALL_OP_COUNT"            $(all_op_count) '=' 3
+# RULE    "OP_COUNT_FC"             $(op_count MATMUL) '=' 1
+# RULE    "FILE_SIZE"               $(file_size) '<' 1040   # As of 191105, file size is 1016
```

Comment:
`# As of 191105, file size is 1016` doesn't seem to be related with this change...

Reply:
`# RULE    "FILE_SIZE" ` may be removed...

Reply:
This is not a part of NNCC 1.0 release requirement ;-D Why don't you use ``circle.rule``?

Reply:
Since the rules are commented out, removing them changes nothing.

Reply:
Maybe it would be better to rename this file, just to `circle.rule` as @parjong 's comment

> This is not a part of NNCC 1.0 release requirement ;-D Why don't you use circle.rule?